src/alto-ui/Datagrid/Datagrid.js

```diff
@@ -462,6 +426,29 @@ class Datagrid extends React.PureComponent {
     );
   }
 
+  renderHorizontalScroll(rowsWidth = 0) {
```

Comment:
so you create your own scrollbar except on macos?

Reply:
yes, on MacOS we don't see a scrollbar. We wan't only scroll by touch pad.